form/bootstrap4.rst

```diff
@@ -0,0 +1,116 @@
+to import Bootstrap's sources into your SCSS and JavaScript files.
+
+The next step is to configure the Symfony application to use Bootstrap 4 styles
+when rendering forms. If you want to apply them to all forms, define this
+configuration:
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # config/packages/twig.yaml
+        twig:
+            form_themes: ['bootstrap_4_layout.html.twig']
+
+    .. code-block:: xml
+
+        <!-- config/packages/twig.xml -->
+        <?xml version="1.0" encoding="UTF-8" ?>
+        <container xmlns="http://symfony.com/schema/dic/services"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xmlns:twig="http://symfony.com/schema/dic/twig"
+            xsi:schemaLocation="http://symfony.com/schema/dic/services
+                http://symfony.com/schema/dic/services/services-1.0.xsd
+                http://symfony.com/schema/dic/twig
+                http://symfony.com/schema/dic/twig/twig-1.0.xsd">
+
+            <twig:config>
+                <twig:form-theme>bootstrap_4_layout.html.twig</twig:form-theme>
+                <!-- ... -->
+            </twig:config>
+        </container>
+
+    .. code-block:: php
+
+        // config/packages/twig.php
+        $container->loadFromExtension('twig', array(
+            'form_themes' => array(
+                'bootstrap_4_layout.html.twig',
+            ),
+
+            // ...
+        ));
+
+If you prefer to apply the Bootstrap styles on a form to form basis, include the
+``form_theme`` tag in the templates where those forms are used:
+
+.. code-block:: twig
+
+    {# ... #}
+    {# this tag only applies to the forms defined in this template #}
+    {% form_theme form 'bootstrap_4_layout.html.twig' %}
+
+    {% block body %}
+        <h1>User Sign Up:</h1>
+        {{ form(form) }}
+    {% endblock body %}
+
+Accessibility
+-------------
+
+The Bootstrap 4 framework has done a good job making it accessible for function
+variations like impaired vision and cognitive ability. Symfony has taken this one
+step further to make sure the form theme complies with the `WCAG2.0 standard`_.
+
+This does not mean that your entire website automatically complies with the full
+standard, but it does mean that you have come far in your work to create a design
+for **all** users.
+
+Custom Forms
+------------
+
+Bootstrap 4 has a feature called "`custom forms`_". You can enable that on your
+Symfony Form ``RadioType`` and ``CheckboxType`` by adding a class called ``radio-custom``
+and ``checkbox-custom`` respectively.
+
+.. code-block:: html+twig
+
+    {{ form_row(form.myRadio, {attr: {class: 'radio-custom'} }) }}
+    {{ form_row(form.myCheckbox, {attr: {class: 'checkbox-custom'} }) }}
+
+Labels and Errors
+-----------------
+
+When you use the Bootstrap form themes and render the fields manually, calling
+``form_label()`` for a checkbox/radio field doesn't show anything. Due to Bootstrap
```

Comment:
`doesn't show anything` -> `doesn't render anything` ?  (and later ... `already shown by` -> `already rendered by` ?)

Reply:
Thank you